api/cookies.go

```diff
@@ -57,3 +69,20 @@ func ExpireSessionCookie() http.Cookie {
 		SameSite: http.SameSiteLaxMode,
 	}
 }
+
+// ExpireIDPSessionCookie expires a cookie for idp
+func ExpireIDPSessionCookie() http.Cookie {
+	return http.Cookie{
+		Path:     "/",
+		Name:     "idp-refresh-token",
+		Value:    "",
+		MaxAge:   -1,
+		Expires:  time.Now().Add(-100 * time.Hour),
```

Comment:
The "official" way to clear the cookie is to set the expiration to epoch (although the result will be the same):
```suggestion
		Expires:  time.Unix(0, 0),
```